FireboltDotNetSdk.Tests/Integration/ExecuteTest.cs

```diff
@@ -62,6 +62,17 @@ private void executeTest(string connString, string commandText, string? addition
             }
         }
 
+        [TestCase("select 1", 1)]
+        [TestCase("select 'hello'", "hello")]
+        public void ExecuteScalar(string query, object expectedValue)
```

Comment:
Let's add another test, where query contains two fields, two rows and verify that `ExecuteScalar` returns only the first row/filed. This way we verify the definition of if from the docs: `Additional columns or rows are ignored.`

Reply:
Done. Take a look again, please